src/cat-data-service/src/service/common/objects/delegator_address.rs

```diff
@@ -0,0 +1,22 @@
+use poem_openapi::{types::Example, Object};
```

Comment:
Every file should have a `//!` comment at the top,

```suggestion
//! Define the Public Key used by a Delegate.
//!
use poem_openapi::{types::Example, Object};
```